docs/appendices/release-notes/unreleased.rst

```diff
@@ -63,6 +63,10 @@ Changes
 - Write blocks added due to low disk space are now automatically removed if a
   node again drops below the high watermark.
 
+- Added a ``WITH`` clause parameter, ``validation`` to ``COPY FROM`` which
```

Comment:
Probably rebase issues? Was added in https://github.com/crate/crate/pull/12219/files

Reply:
This entry is already on master (see PR above) and should not appear in changes after successful rebase.